jetty-ee9/jetty-ee9-websocket/jetty-ee9-websocket-jakarta-server/src/main/java/org/eclipse/jetty/ee9/websocket/jakarta/server/config/JakartaWebSocketConfiguration.java

```diff
@@ -35,6 +35,6 @@ public JakartaWebSocketConfiguration()
         protectAndExpose("org.eclipse.jetty.ee9.websocket.servlet."); // For WebSocketUpgradeFilter
         protectAndExpose("org.eclipse.jetty.ee9.websocket.jakarta.server.config.");
         protectAndExpose("org.eclipse.jetty.ee9.websocket.jakarta.client.JakartaWebSocketClientContainerProvider");
-        protectAndExpose("org.eclipse.jetty.ee9.websocket.jakarta.client.JakartaWebSocketShutdownContainer");
+        protectAndExpose("org.eclipse.jetty.ee9.websocket.jakarta.client.internal.JakartaWebSocketShutdownContainer");
```

Comment:
Feels wrong that we protect and expose an internal class, and not sure it really works in JPMS.
I think it's best to make it non-internal like the other one.

Reply:
Perhaps we can look at this in another PR, but I think it should be possible to look for SCIs in the environment classloader, that are not exposed to the webapp classloader.